src/test/java/net/minecraftforge/debug/client/rendering/RenderableTest.java

```diff
@@ -30,19 +36,23 @@
 import net.minecraftforge.client.model.renderable.IRenderable;
 import net.minecraftforge.client.model.renderable.MultipartTransforms;
 import net.minecraftforge.common.MinecraftForge;
-import net.minecraftforge.eventbus.api.Event;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
 import net.minecraftforge.fml.loading.FMLEnvironment;
 
 import java.util.Map;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 @Mod(RenderableTest.MODID)
 public class RenderableTest
```

Comment:
Please provide a javadoc for this class which explains what feature this tests, how to test it, and what the expected results are from said test. 

As you are only modifying an existing test mod, you need only include what is relevant to this PR, with an note about there being other behaviors tested by this mod which aren't documented yet.